packages/@blockprotocol/type-system-node/tests/uri.test.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseUri, VersionedUri } from "@blockprotocol/type-system-node";
```

Comment:
Might be fixable via

```suggestion
import { BaseUri, VersionedUri } from "..";
```

Reply:
As discussed offline, this was a misguided attempt to test the `package.json`.
Changed in https://github.com/blockprotocol/blockprotocol/pull/503/commits/d93057ed98481477f6feeee5c1072c988dc578ca